patterns/src/main/java/pattern/creation/abstractfactory/Application.java

```diff
@@ -0,0 +1,26 @@
+package pattern.creation.abstractfactory;
+
+public class Application {
+    private OrderCoffeeForm orderCoffeeForm;
+
+    public void drawOrderCoffeeForm() {
+        // Определим имя операционной системы, получив значение системной проперти через System.getProperty
+        String osName = System.getProperty("os.name").toLowerCase();
+        GUIFactory guiFactory;
+
+        if (osName.startsWith("win")) { // Для windows
```

Comment:
@phx67  create Enum with os.name and change 'if' to 'switch' or create a method with an input value and return Enum !! If you do not understand please ping me !!!!!